core/src/avm1/object.rs

```diff
@@ -60,6 +58,7 @@ pub enum NativeObject<'gc> {
     TextFormat(GcCell<'gc, TextFormat>),
     NetStream(NetStream<'gc>),
     BitmapData(BitmapDataWrapper<'gc>),
+    XmlNodeObject(XmlNode<'gc>),
```

Comment:
The naming convention here is that this should be `XmlNode` instead of `XmlNodeObject`.

Reply:
Yep sorry copy paste error, fixed!